autoload/neomake.vim

```diff
@@ -248,10 +248,12 @@ function! s:Make(options) abort
         endif
     endif
 
+    let title = join(map(copy(enabled_makers), 'v:val.name'), ', ')
+
     if file_mode
-        lgetexpr ''
+        call setloclist([], 'r', title)
     else
-        cgetexpr ''
+        call setqflist([], 'r', title)
```

Comment:
Please handle `E118` for backwards compatibility: https://github.com/neovim/neovim/pull/4337#issuecomment-188891314